### PR TITLE
Add documentation for app network migration

### LINKED ISSUE #81: Mobile devs SBAT follow documentation on how to migrate to a new network
### Expected Behavior

Mobile developers can follow documentation that lays out steps to make the mobile app compatible with a new network. These steps include updating contract addresses in `.env` files, updating and deploying the blockchain-api, and updating the notification service. 

### Current Behavior

Mobile developers cannot reference documentation on migrating the app to a new network, so they ask @cmcewen for help.

### PR DESCRIPTION
**Pull request by annakaz**
### Description

Documentation listing changes required on app side to migrate to a new network. Steps include updating contract addresses in .env files, updating the notification service, and deploying the blockchain-api.

### Tested

Went through steps to migrate app to `pilot`

### Related issues

- Fixes #81

### CODE REVIEW

File: packages/mobile/README.md [hunk 1]
```diff
@@ -88,6 +88,18 @@ that you want to use with the app, you can run
 yarn run build-sdk TESTNET
 ```
 
+## Migrating networks
+
+To migrate to a new network, update the environment configuration files within the app and its dependencies:
+
+Update `.env.ENV-NAME` and `packages/mobile/.env.ENV-NAME` with the new network name and settings. 
+
+For the blockchain api, update `packages/blockchain-api/app.ENV-NAME.yaml` with the new contract addresses, which can be found using `celotooljs` or by looking at the address of the smart contracts in `/packages/protocol/build/ENV-NAME/contracts`.
```
Comment (jmrossy) [inline packages/mobile/README.md]: Appreciate the update here but not sure this is the best place to have details about blockchain api and notification service.
1. Don't think these details are relevant to external devs
2. This should probably live in a bigger "How to bring up a testnet" doc :)

Comment (annakaz) [reply packages/mobile/README.md]: Great point! Can you link the how to bring up a testnet doc?

Comment (jmrossy) [reply packages/mobile/README.md]: It needs work but here's what we've got so far https://docs.google.com/document/d/1vMXZY6pu2fmwHxTzZE0YbKhn6Gs9szPZjHtjvV8dsis/edit#heading=h.ybn9lro53cxv

**Comment by annakaz** [comment]
Updated Readme with externally relevant content, put blockchain-api and notification-service details in  "How to bring up a testnet" doc

**Review by jmrossy — DISMISSED** [review]

**Review by cmcewen — APPROVED** [review]

**Review by tkporter — APPROVED** [review]

**Pull request merged**